### PR TITLE
This commit adds the monster fighter MVP

### PR DESCRIPTION
**Pull request by mikehiett-mpb**

### CODE REVIEW

File: entities/fights.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            if actor.health <= 0:
+                print('The monster killed you!')
+                return True
+        return False
+
+    def get_prompt(self):
+        return self.question
+
+    def get_available_actions(self):
+        return ['Blow bubbles at', 'Hit', 'Kick']
```
Comment (sophiedaviespatrick) [inline entities/fights.py]: (Practising git reviews) Suggest you change the first action so it is quicker for the user to type e.g. "Blow bubbles" without the "at"

**Review by sophiedaviespatrick — CHANGES_REQUESTED** [review]

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (chasinglogic) [inline .idea/inspectionProfiles/profiles_settings.xml]: Need to remove these `.idea` files

**Review by chasinglogic — APPROVED** [review]
As long as the .idea files get removed this LGTM.

File: entities/fights.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+from entities.entity import Entity
+
+
+class Fight(Entity):
+    def __init__(self):
+        self.question = "You're in the fight what do you want to do ?"
+
+    def do(self, action, actor, monster):
```
Comment (2Steaks) [inline entities/fights.py:8]: There does not seem to be anything to handle "Kick"

Comment (2Steaks) [inline entities/fights.py]: There does not seem to be anything to handle "Kick" or "Blow bubbles"

File: entities/monster.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class Monster(Entity):
+    def __init__(self):
+        self.health = 20
+        self.question = "You find a monster!! What do you want to do?"
+
+    def do(self, action, actor):
+        if action == 'Run':
+            return True
```
Comment (2Steaks) [inline entities/monster.py]: Might be good to have a random outcome here, where running might be unsuccessful and maybe you lose some health as a penalty before entering the fight?

File: entities/fights.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        if action == 'Hit':
+            monster.health -= 5
```
Comment (2Steaks) [inline entities/fights.py]: Maybe randomise damage for all attacks

**Review by 2Steaks — CHANGES_REQUESTED** [review]
Missing fight action handlers

File: entities/fights.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+            print(f'You hit the monster for 5 health! It now has {monster.health} health left')
+            if monster.health <= 0:
+                print('You killed the monster!')
+                return True
+            actor.health -= 5
+            print(f'The monster hits you back for 5 health! You now have {actor.health} health left!')
```
Comment (davidttest) [inline entities/fights.py]: Do we want the monster to hit back every time if we've hit him first?

File: entities/monster.py (hunk 4 above)
Comment (davidttest) [reply entities/monster.py]: Agree, courage should prevail! :)

**Review by davidttest — CHANGES_REQUESTED** [review]
Should probably handle kick and blow bubbles, a couple of comments about nice to haves

File: entities/fights.py (hunk 1 above)
Comment (mikehiett-mpb) [reply entities/fights.py]: Done!

File: entities/fights.py (hunk 5 above)
Comment (chasinglogic) [reply entities/fights.py]: Not in MVP

File: entities/fights.py (hunk 6 above)
Comment (chasinglogic) [reply entities/fights.py]: Not in MVP

File: entities/fights.py (hunk 3 above)
Comment (chasinglogic) [reply entities/fights.py]: These actions are always intended to miss for MVP

File: entities/monster.py (hunk 4 above)
Comment (chasinglogic) [reply entities/monster.py]: Not in MVP

**Pull request closed without merging**